news/11099.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fix reliance on deprecated stdlib cgi module.
```

Comment:
```suggestion
Remove reliance on the stdlib cgi module, which is deprecated in Python 3.11.
```

This should be a `process` fragment IMO.

Reply:
Thanks! I'll do that instead.